PhysicsTools/Heppy/python/analyzers/gen/PDFWeightsAnalyzer.py

```diff
@@ -18,8 +18,9 @@ def __init__(self, cfg_ana, cfg_comp, looperName ):
     def declareHandles(self):
         super(PDFWeightsAnalyzer, self).declareHandles()
 
-        if self.doPDFWeights:
-            self.mchandles['pdfstuff'] = AutoHandle( 'generator', 'GenEventInfoProduct' )
+        #if self.doPDFWeights:
```

Comment:
I would add a flag "doX1X2etc" and put in or for the doPDFWeights... even if it is not clear to me if there is any use of this analyzer without reading that...